### ISSUE TITLE
Unable to install package firetv>=1.0.6: Failed building wheel for pycryptodome

### ISSUE THREAD

**Issue by gsbrown** [JeffLIrion/homeassistant_native_firetv#16]
```
2018-10-29 16:52:45 ERROR (SyncWorker_0) [homeassistant.util.package] Unable to install package firetv>=1.0.6: Failed building wheel for pycryptodome
python-jose-cryptodome 1.3.2 has requirement pycryptodome<3.4.0,>=3.3.1, but you'll have pycryptodome 3.7.0 which is incompatible.
pyrebase4 4.2.0 has requirement oauth2client==4.1.2, but you'll have oauth2client 4.0.0 which is incompatible.
broadlink 0.9 has requirement pycryptodome==3.4.11, but you'll have pycryptodome 3.7.0 which is incompatible.
Command "/usr/local/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-l8w5rut0/pycryptodome/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-pa4uey6n/install-record.txt --single-version-externally-managed --prefix  --compile --user --prefix=" failed with error code 1 in /tmp/pip-install-l8w5rut0/pycryptodome/
You are using pip version 18.0, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
2018-10-29 16:52:45 ERROR (MainThread) [homeassistant.requirements] Not initializing media_player.firetv because could not install requirement firetv>=1.0.6
2018-10-29 16:52:45 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform media_player.firetv: Could not install all requirements.
```
Recently updated Hassio and found that the firetv server I was using no longer worked. I stumbled across the native version but am running in to the above errors. I am on Hassio 0.81.1; appreciate any assistance.

**Comment by JeffLIrion** [comment]
My guess is that you're using some other custom components that create a version conflict for `pycryptodome`. Please try removing all other custom components and see if the error persists. (If this is the cause, you might get the same sort of error when you reintroduce the other custom components.)

**Comment by gsbrown** [comment]
Didn't work. I cleared everything else out. I am running hassio on an ubuntu server box - if that makes a difference somehow?

**Comment by JeffLIrion** [comment]
You got the same error? That's odd because it HA should enforce that the version of `pycryptodome` is >=3.6.6 (https://github.com/home-assistant/home-assistant/pull/16238)

**Comment by gsbrown** [comment]
Yes, there was no change to the error. I took everything else out of the custom_components folder. Sorry I'm not much more help, I haven't messed around in my HA setup for quite some time.

**Comment by JeffLIrion** [comment]
In your configuration folder, there should be a folder named `deps`. Try renaming this to, say, `deps_BACKUP` and see if that fixes the issue.

**Comment by gsbrown** [comment]
No dice, was hoping that would work! I am at a loss - I was hoping it was just due to a hassio update, breaking change or something?

**Comment by JeffLIrion** [comment]
Are you sure you're using Hass.io? I think that maybe you installed Home Assistant using your system's Python3. If that's the case, I think you have 2 options:

1. Sort out the version conflicts yourself. 
2. Install Home Assistant in a [virtual environment](
https://www.home-assistant.io/docs/installation/virtualenv/)

Assuming this is the case, I recommend option 2.

**Comment by JeffLIrion** [comment]
I'm going to close this issue, as it seems to be more related to your overall HA setup environment than to this component. You can re-open or re-submit it if you wish.

**Comment by photinus** [comment]
I just randomly hit this issue today as well. I have two other custom components, neither of which use pycryptodome. 

Currently running Home Assistant 0.82.1
Other custom components are the Alexa TTS and a custom copy of the native Vizio component.

**Comment by JeffLIrion** [comment]
According to issue https://github.com/JeffLIrion/homeassistant_native_firetv/issues/19, this is resolved in the latest beta version (0.83.0b1) and will be resolved in the next release (0.83).
